re/ReForm_Validation.re

```diff
@@ -3,25 +3,59 @@
 module I18n = {
   type dictionary = {
     required: string,
-    email: string
+    email: string,
   };
   let ptBR = {
     required: {j|Campo obrigatório|j},
-    email: {j|Email inválido|j}
+    email: {j|Email inválido|j},
   };
   let en = {required: "Field is required", email: "Invalid email"};
 };
 
+type error = option(string);
+
+type errors = option(list(string));
+
 type validation('values) =
   | Required
   | Email
-  | Custom('values => option(string));
+  | Custom('values => error);
+
+/* turn list(error) => errors */
+let squeeze = (errorList: list(error)) => {
```

Comment:
hmm I think we just should return the list of errors to the user

Reply:
Do you mean list(string) ?